tsdb/head_test.go

```diff
@@ -5542,3 +5542,54 @@ func TestHeadCompactionWhileAppendAndCommitExemplar(t *testing.T) {
 	app.Commit()
 	h.Close()
 }
+
+func TestStripeSeries_gc(t *testing.T) {
+	lbls1 := labels.FromMap(map[string]string{
+		"__name__": "metric",
+		"lbl":      "qeYKm3",
+	})
+	hash := lbls1.Hash()
+	ms1 := memSeries{
+		lset: lbls1,
+	}
+	lbls2 := labels.FromMap(map[string]string{
+		"__name__": "metric",
+		"lbl":      "2fUczT",
```

Comment:
I think this is worth a comment how these values were chosen, and also which version of `Labels.Hash` this is true for.  
Perhaps like  https://github.com/grafana/mimir/blob/8f2a074d3bc4049cdf815e16cf37660e8b85338c/pkg/ingester/activeseries/active_series_test.go#L436

Reply:
Done.